README.md

```diff
@@ -16,32 +16,51 @@ environment:
 configmap:
   VAR_A: VALUE_A
   VAR_B: VALUE_B
+secrets: 
+  ENVIRONMENT_VAR:
+      secretRef: some-secret-reference
+      key: connectionString
 ```
 
+### secrets
+Secret objects in this configuration consist of name secretRef and key. These need to be put into the secrets array in the following manor
+```yaml
+secrets: 
+  ENVIRONMENT_VAR:
+      secretRef: some-secret-reference
+      key: connectionString
+```
+*Where:*
+- **ENVIRONMENT_VAR** is the environment variable name to reference ( this has to be capitals and can contain numbers or "_" ).
+- **secretRef** is the service instance ( as in the case of PaaS wrappers ) or reference to the secret volume
```

Comment:
secret volume or just 'secret'?

Reply:
well its for the connected service instance in our case that looks like an attached volume I believe.

Reply:
a secret is a secret no matter the implementation.  A n00b reading that might think they need to have a 'secret volume' or a 'PaaS wrapper'.  Best to keep the docs generic, because that's how you've (correctly) implemented it :-)